hubstorage/resourcetype.py

```diff
@@ -26,7 +28,9 @@ def _iter_lines(self, _path, **kwargs):
 
         r = self.client.request(**kwargs)
 
-        return r.iter_lines()
+        lines = r.iter_lines()
+        return (l.decode('utf8') if isinstance(l, six.binary_type)
+                else l for l in lines)
```

Comment:
About `iter_lines(decode_unicode=True)`, the response coming from Hubstorage has no encoding sett, so `requests` library doesn't decode the data ¹.

¹https://github.com/kennethreitz/requests/blob/e65d6fc6378826e70304b7db660ab37d306dff54/requests/utils.py#L362